CHANGELOG.md

```diff
@@ -2,6 +2,36 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
+## v0.3.0 Venice
+
+The parser and semantic analysis stage of Solang have been through
```

Comment:
I think `have gone through` or `underwent` sounds better than `have been through`.

Reply:
I've changed to `have gone through` but I have a hard time seeing any difference - other than taste.